samigo/samigo-app/src/webapp/jsf/questionpool/previewPool.jsp

```diff
@@ -0,0 +1,148 @@
+<%@ page contentType="text/html;charset=utf-8" pageEncoding="utf-8" language="java" %>
+<%@ taglib uri="http://java.sun.com/jsf/html" prefix="h" %>
+<%@ taglib uri="http://java.sun.com/jsf/core" prefix="f" %>
+<%@ taglib uri="http://www.sakaiproject.org/samigo" prefix="samigo" %>
+<!DOCTYPE html
+     PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
+     "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+
+  <f:view>
+    <html xmlns="http://www.w3.org/1999/xhtml">
+      <head><%= request.getAttribute("html.head") %>
+      <title><h:outputText value="#{questionPoolMessages.t_previewPool}"/></title>
+                        <!-- stylesheet and script widgets -->
+<script language="javascript" type="text/JavaScript">
+<!--
+<%@ include file="/js/samigotree.js" %>
+//-->
+</script>
+      </head>
+<body onload="<%= request.getAttribute("html.body.onload") %>">
+<!-- content... -->
+ <div class="portletBody">
+
+<h:messages infoClass="validation" warnClass="validation" errorClass="validation" fatalClass="validation"/>
+
+<h3 class="insColor insBak insBor">
+<h:outputText value="#{questionPoolMessages.qp}#{questionPoolMessages.column} #{questionpool.currentPool.displayName}"/>
+</h3>
+
+<div class="tier1">
+<h:form id="previewPool">
+<p class="navViewAction">
+<h:commandLink title="#{questionPoolMessages.t_export}" rendered="#{questionpool.importToAuthoring != 'true'}" id="exportlink" immediate="true" action="#{questionpool.exportPool}">
+  <h:outputText id="exportq" value="#{questionPoolMessages.export}"/>
+  <f:param name="poolId" value="#{questionpool.currentPool.id}"/>
+</h:commandLink>
+<h:outputText  rendered="#{questionpool.importToAuthoring != 'true'}" value=" #{questionPoolMessages.separator} " />
+<h:outputLink id="printlink" rendered="#{questionpool.importToAuthoring != 'true'}" onclick="print(); return false;" title="#{questionPoolMessages.t_print}" >
+	<h:outputText id="printq" value="#{questionPoolMessages.print}"/>
+</h:outputLink>
+</p>
+</h:form>
+</div>
+
+
+<h:dataTable id="parts" width="100%"
```

Comment:
I'd prefer to see these in-line styles off-loaded to an appropriate CSS file. Same with below.